src/index.html

```diff
@@ -1,13 +1,323 @@
+              Національний художній музей України до 100 річчя від дня народження   видатної української художниці  Тетяни Яблонської  відкриває ретроспективну ювілейну виставку «І спогади і мрії».
+            </p>
+          </div>
+        </div>
+
+        <div class="events__content lecture">
+          <img
+          src="./images/events-3-desktop.png"
+          alt="lecture"
+          class="lecture__image"
+          >
+          <h2 class="event__subtitle lecture__subtitle">
+            лекція
+            <p class="event__subtitle-date lecture__subtitle-date">
+              26/08/2019 - 11:00
+            </p>
+            <div class="lecture__title">
+              Від класицизму до романтизму
+            </div>
+          </h2>
+        </div>
+      </section>
+
+      <section class="page__gallery gallery" id="gallery">
+        <h1 class="gallery__title">
+          Галерея
+        </h1>
+        <div class="gallery__content">
+          <div class="gallery__image gallery__image-first">
+            <img
+              src="./images/gallery-1.png"
+              alt="gallery-1"
+              class="gallery__img"
+            >
+          </div>
+          <div class="gallery__image gallery__image-second">
+            <img
+              src="./images/gallery-2.png"
+              alt="gallery-2"
+              class="gallery__img"
+            >
+          </div>
+          <div class="gallery__image gallery__image-third">
+            <img
+              src="./images/gallery-3.png"
+              alt="gallery-3"
+              class="gallery__img"
+            >
+          </div>
+          <div class="gallery__image gallery__image-fourth">
+            <img
+              src="./images/gallery-4.png"
+              alt="gallery-4"
+              class="gallery__img"
+            >
+          </div>
+          <div class="gallery__image gallery__image-fifth">
+            <img
+              src="./images/gallery-5.png"
+              alt="gallery-5"
+              class="gallery__img"
+            >
+          </div>
+        </div>
+        <div class="gallery__selector">
+          <img src="./images/ellipse-active.png" alt="1">
+          <img src="./images/ellipse.png" alt="0">
+          <img src="./images/ellipse.png" alt="0">
+          <img src="./images/ellipse.png" alt="0">
+        </div>
+      </section>
+
+      <section class="page__subscription subscription" id="subscription">
+        <h2 class="subscription__title">
+          Підписка
+        </h2>
+        <p class="subscription__description">
+          Бути в курсі всіх актуальних подій та останніх новин.
+        </p>
+        <form
+          action="#"
+          method="get"
+          class="subscription__form form form--sub"
+        >
+          <input
+            type="text"
+            class="form-input"
+            placeholder="E-mail"
+            required
+          >
+          <input
+            type="reset"
```

Comment:
use js for [resetting](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset)